source/hassio/debugging.markdown

```diff
@@ -13,7 +13,7 @@ The following debug tips and tricks are for people who are running the Hass.io i
 
 ## {% linkable_title SSH access to the host %}
 
-Create an `authorized_keys` file in the root of your SD card with your public key. Once the device is booted, you can access your device as root over SSH on port 22222.
+Create an `authorized_keys` file in the root of your SD card with your public key. Once the device is booted, you can access your device as root over SSH on port 22222. The contents of the 'authorized_keys' file should be the contents of your 'id_rsa.pub' file.
```

Comment:
Sometimes these keys are created by third party subscriptions e.g. github(.rsa.pub). many users now 'subscribe' using their github and/or other public keys.